platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/Checkstyle.java

```diff
@@ -271,6 +279,7 @@ public DirectoryProperty getConfigDirectory() {
      */
     @Override
     @Nested
+    @ToBeKeptEagerProperty(because = "Read-only nested property")
```

Comment:
:x: We should skip methods already annotated with `@Nested`, no?

Reply:
Done